### ISSUE TITLE
[7.4.0] Error at resize of window after update to Chrome 80

### ISSUE THREAD

**Issue by nethore** [handsontable/handsontable#6710]
### Description
Hi all,
I've a problem that appears with latest Chrome 80 update. Now, when I resize my window, the handsontable table is not stretching by itself to fill the div.

Error in console is the following:
```
zone.js:199 Uncaught TypeError: Cannot read property 'nodeType' of undefined
    at closest (element.js:106)
    at extendEvent (eventManager.js:301)
    at callbackProxy (eventManager.js:65)
```

### Steps to reproduce
1. Simply resize the browser's window.

### Demo
Cannot reproduce, will try again on Monday

### Your environment
* Handsontable version: 7.4.0 - Angular wrapper
* Browser Name and version: Chrome 80 (work well with 79)
* Operating System: Mac OS Catalina / Also tested on Windows 10

**Comment by nethore** [comment]
Good to hear you find out where the problem was coming from 🎆.
Is there any release planned to fix it ?

**Comment by wojciechczerniak** [comment]
Yes, we want to release 7.4.1 ASAP, the exact day depends on the review & testing process.

Thank you very much for reporting the issue @nethore

**Comment by aninde** [comment]
@nethore this bug will be fixed by 7.4.1 v. We plan to release it soon.

I tested it locally by comparing versions `7.4.0 v.` and `7.4.1 v.` on latest `Chrome 80.0.3987.106`.

**7.4.0 v. with Angular wrapper 5.1.0**
![7 4 0-chrome80](https://user-images.githubusercontent.com/30186818/74740521-4f0d6180-525b-11ea-9425-34ba6737091c.gif)

**Fixed 7.4.1 v. with Angular wrapper 5.1.0**
![7 4 1-chrome80](https://user-images.githubusercontent.com/30186818/74740522-50d72500-525b-11ea-8714-760b7e09d48f.gif)

**Comment by AMBudnik** [comment]
Hey @nethore 

I hope it works for you well. We have officially fixed this issue in `7.4.2`. If there are any other issues please feel free to open a new ticket.

**Comment by dovk** [comment]
Thanks alot! Moved from 7.0.2 to 7.4.2 and it's OK!
